packages/expo/src/launch/withDevTools.tsx

```diff
@@ -0,0 +1,53 @@
+import Constants, { ExecutionEnvironment } from 'expo-constants';
+import * as React from 'react';
+import { Platform } from 'react-native';
+
+import DevLoadingView from '../environment/DevLoadingView';
+
+/**
+ * Append the Expo Fast Refresh view and optionally
+ * keep the screen awake if a `expo-keep-awake` is installed.
+ */
+export function withDevTools<TComponent extends React.ComponentType<any>>(
+  AppRootComponent: TComponent
+): React.ComponentType<React.ComponentProps<TComponent>> {
+  // This hook can be optionally imported because __DEV__ never changes during runtime.
+  // Using __DEV__ like this enables tree shaking to remove the hook in production.
+  const useOptionalKeepAwake: (tag?: string) => void = (() => {
+    if (Platform.OS !== 'web') {
+      try {
+        // Optionally import expo-keep-awake
+        const { useKeepAwake, ExpoKeepAwakeTag } = require('expo-keep-awake');
+        return () => useKeepAwake(ExpoKeepAwakeTag, { suppressDeactivateWarnings: true });
+      } catch {}
+    }
+
+    return () => {};
+  })();
+
+  const useExpoFastRefreshView = Platform.select({
```

Comment:
Name this differently than a hook:
```suggestion
  const shouldUseExpoFastRefreshView = Platform.select({
```

Reply:
^